Xcode/LiteCore.xcodeproj/project.pbxproj

```diff
@@ -432,10 +432,10 @@
 			fileType = sourcecode.java;
 			isEditable = 1;
 			outputFiles = (
-				"$(DERIVED_FILE_DIR)/Java_com_couchbase_litecore_Database.h",
+				"$(DERIVED_FILE_DIR)/Java_com_couchbase_lite_internal_Database.h",
```

Comment:
I'm not aware of generating JNI header in XCode project and I don't know why there is only Database.h.

@borrrden?

Reply:
This was added 3 years ago by @snej for reasons that may or may not still be relevant ( see https://github.com/couchbase/couchbase-lite-core/commit/3ef6a76fb2532f18f50607624e1af69f7e78f124 )